schemas/context/account-crm.schema.json

```diff
@@ -0,0 +1,159 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/account-crm",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "XDM Business Account",
+  "type": "object",
+  "auditable": true,
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": [
+    "https://ns.adobe.com/xdm/context/account"
+  ],
+  "description": "An XDM Business Account.",
+  "definitions": {
+    "account-crm": {
+      "properties": {
+        "xdm:ownerID": {
+          "title": "Account Owner",
+          "description": "assigned owner of account. Not available in personal edition.",
+          "type": "string"
+        },
+        "xdm:parentID": {
+          "title": "Parent Account",
+          "description": "Parent company for companies that are subsidiaries of a larger company or organization. The parent account must be an existing account in salesforce. You can enter the account name, or select (or optionally, create) the account using the lookup icon.",
+          "type": "string"
+        },
+        "xdm:partnerAccountID": {
+          "title": "Partner Account Id",
+          "description": "Partner account id.",
+          "type": "string"
+        },
+        "xdm:masterRecordID": {
```

Comment:
What's the significance of MasterRecordID?

Reply:
Removed